sources/packages/backend/apps/db-migrations/src/sql/SFASApplications/Add-data-expansion-columns.sql

```diff
@@ -0,0 +1,102 @@
+ALTER TABLE
+    sims.sfas_applications
+ADD
+    COLUMN application_number INT,
+ADD
+    COLUMN living_arrangements CHAR(1),
+ADD
+    COLUMN marital_status VARCHAR(4),
+ADD
+    COLUMN marriage_date DATE,
+ADD
+    COLUMN bc_residency_flag CHAR(1),
+ADD
+    COLUMN permanent_residency_flag CHAR(1),
+ADD
+    COLUMN gross_income_previous_year INT,
+ADD
+    COLUMN institution_code CHAR(4),
+ADD
+    COLUMN application_status_code CHAR(4),
+ADD
+    COLUMN education_period_weeks INT,
```

Comment:
Do we need a 4-byte int to save columns like `education_period_weeks` and `course_load`?

Reply:
I will set to smallint.